keps/sig-scheduling/5004-dra-extended-resource/kep.yaml

```diff
@@ -0,0 +1,47 @@
+title: DRA Extended Resource
+kep-number: 5004
+authors:
+  - "@yliaog"
+owning-sig: sig-scheduling
+participating-sigs:
+  - sig-node
+status: draft
+creation-date: 2025-02-03
+reviewers:
+  - "@klueska"
+  - "@johnbelamaric"
+  - "@pohly"
+approvers:
+  - "@mrunalp" # SIG-Node
+  - "@alculquicondor" # SIG-Scheduling
+  - "@thockin" # API Review
+
+see-also:
+  - "/keps/sig-node/4381-dra-structured-parameters"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.33"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.33"
```

Comment:
should be 1.34 and also update the beta and stable ones

Reply:
fixed